v5/tutorials/topical/multitasking.rst

```diff
@@ -59,7 +59,7 @@ Tasks in PROS are simple to create:
                 // ...
             }
             void initialize() {
-                task_t my_task = task_create(my_task_fn, "PROS", TASK_PRIORITY_DEFAULT,
+                task_t my_task = task_create(my_task_fn, (void*)"parameter(s) here", TASK_PRIORITY_DEFAULT,

```

Comment:
In c, there's an implicit conversion to a void* and back. Hello parameter(s) here also isn't quite right. Maybe pass some information by a struct? It's something I see people asking about occasionally.

Reply:
might also want to showcase that most of these parameters are unnecessary. lots of people put them in for no reason other than that they're in the tutorial